changelogs/fragments/bugfix_100.yaml

```
@@ -0,0 +1,3 @@
+---
+bugfixes:
```

This is a non-user facing change.
```suggestion
trivial:
```